pkg/operator/ceph/controller/spec_test.go

```diff
@@ -382,63 +382,63 @@ func TestLogCollectorContainer(t *testing.T) {
 	t.Run("Periodicity 1d and no MaxlogSize", func(t *testing.T) {
 		c := cephv1.ClusterSpec{LogCollector: cephv1.LogCollectorSpec{Enabled: true, Periodicity: "1d"}}
 		got := LogCollectorContainer(daemonId, ns, c)
-		want := fmt.Sprintf(cronLogRotate, daemonId, "daily", "0")
+		want := fmt.Sprintf(cronLogRotate, daemonId, "daily", "0", "7")
 		assert.Equal(t, want, got.Command[5])
 	})
 
 	t.Run("Periodicity 1h and MaxlogSize 1M", func(t *testing.T) {
 		maxsize, _ := resource.ParseQuantity("1M")
 		c := cephv1.ClusterSpec{LogCollector: cephv1.LogCollectorSpec{Enabled: true, Periodicity: "1h", MaxLogSize: &maxsize}}
-		got := LogCollectorContainer(daemonId, ns, c)
-		want := fmt.Sprintf(cronLogRotate, daemonId, "hourly", "1M")
+		got := LogCollectorContainer("*-client.rbd-mirror*", ns, c)
```

Comment:
I'd use an actual string (something like `ceph-client.rbd-mirror.a`) instead of the mask.

Reply:
done